pkg/apis/core/typeconfig/util.go

```diff
@@ -51,3 +54,22 @@ func GroupQualifiedName(apiResource metav1.APIResource) string {
 	}
 	return fmt.Sprintf("%s.%s", apiResource.Name, apiResource.Group)
 }
+
+func NamespacedToScope(apiResource metav1.APIResource) apiextv1b1.ResourceScope {
```

Comment:
Please move this function and the one that follows to the enable package (whether `enable.go` or `util.go`), since that's the only place they are used.  All other code can just use the namespace value of the apiresource verbatim and won't have to consider the discrepancy.

Reply:
Fixed.